sd/etcd/integration_test.go

```diff
@@ -0,0 +1,122 @@
+// +build integration
+
+package etcd
+
+import (
+	"flag"
+	"fmt"
+	"kit/log"
+	"os"
+	"testing"
+	"time"
+
+	etcdc "github.com/coreos/etcd/client"
+	etcdi "github.com/coreos/etcd/integration"
+	"golang.org/x/net/context"
+)
+
+var (
+	host             []string
+	kitClientOptions ClientOptions
+)
+
+func TestMain(m *testing.M) {
+	flag.Parse()
+
+	fmt.Println("Starting etcd server...")
```

Comment:
I dont think so. I just noticed something similar in the zookeeper package. Ill remove.